documentation-site/static/examples/tag/clickable-non-closeable.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {Tag, KIND, VARIANT} from 'baseui/tag';
+
+const kinds = ['neutral', 'primary', 'positive', 'warning', 'negative'];
```

Comment:
should we use `Object.keys(KIND)` instead, so we don't have to update this example is KINDs are added?

Reply:
For sure, I just didn't want to add a `custom` example here